pkg/apis/core/types.go

```diff
@@ -1741,7 +1741,7 @@ type CSIPersistentVolumeSource struct {
 	// ControllerExpandSecretRef is a reference to the secret object containing
 	// sensitive information to pass to the CSI driver to complete the CSI
 	// ControllerExpandVolume call.
-	// This is an beta field and requires enabling ExpandCSIVolumes feature gate.
+	// This is a beta field.
```

Comment:
Is this still beta or GA? If beta, it should still mention the feature gate that guards it

Reply:
I took a look at [the KEP](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/556-csi-volume-resizing#supporting-per-pvc-secret-refs) and [the PR on which this field was added](https://github.com/kubernetes/kubernetes/commit/69393291b64a2281533cb2567f67edca478b4a93). This field is controlled by the `ExpandCSIVolumes` function gate, so it has reached GA.
```
This API change will be gated by ExpandCSIVolumes feature gate
```

I will remove the content: "This is a beta field."